### ISSUE TITLE
b3d gcloud cp errors, should use rsync instead of cp

### ISSUE THREAD

**Issue by sritchie** [probcomp/b3d#93]
After some poking around, this command seems to be the money:

```sh
gsutil -m rsync -r -x ".*\.gstmp$" gs://b3d_bucket /home/sritchie/b3d/assets/shared_data_bucket
```

I'll send a PR soon that suggests this change for b3d_pull.

**Comment by sritchie** [comment]
First and second run:

```sh
\ [130/130 files][ 28.2 GiB/ 28.2 GiB] 100% Done  67.2 MiB/s ETA 00:00:00
Operation completed over 130 objects/28.2 GiB.

Success: Downloaded all contents from https://console.cloud.google.com/storage/browser/b3d_bucket.

real	1m48.090s
user	6m52.650s
sys	3m50.464s
sritchie@sam-b3d-repo:~/b3d$ time b3d_pull
B3D Downloading data from gs://b3d_bucket to /home/sritchie/b3d/assets/shared_data_bucket.
===============================================
Downloading all contents from gs://b3d_bucket into /home/sritchie/b3d/assets/shared_data_bucket...
(overwrite=False)
===============================================
Building synchronization state...
Starting synchronization...

Success: Downloaded all contents from https://console.cloud.google.com/storage/browser/b3d_bucket.

real	0m8.733s
user	0m12.942s
sys	0m4.672s
```

**Comment by sritchie** [comment]
Closed by #94
